### PR TITLE
engineering: the fundamental of web performance

### PR DESCRIPTION
**Pull request by zlatanpham**

### CODE REVIEW

**Review by monotykamary — APPROVED** [review]
LGTM, although maybe tags are a bit too general.

**Comment by trankhacvy** [comment]
Agree, the tags and the content are a bit general. Could we consider having serial notes for the specific factor of web performance instead of a general one?
Some examples: 
- https://web.dev/learn-core-web-vitals/
- https://web.dev/fast/

File: Engineering/Frontend/The fundamental of web performance.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
+---
+tags: engineering/frontend, frontend, frontend/performance, performance, fundamental
+author: Pham Duc Thanh
+date: 2022-02-10
```
Comment (janrvu) [inline Engineering/Frontend/The fundamental of web performance.md:4]: Should be formatted to YYYY-MM-DD => 2022-10-02

**Review by janrvu — CHANGES_REQUESTED** [review]
Please change the date format to YYYY-MM-DD

**Comment by tienan92it** [comment]
LGTM

**Review by tienan92it — APPROVED** [review]

**Comment by zlatanpham** [comment]
> Agree, the tags and the content are a bit general. Could we consider having serial notes for the specific factor of web performance instead of a general one?

Yes, we will have other linked notes for those specific topics. For example, I'm currently writing a note on resource hint.

**Comment by zlatanpham** [comment]
And about the general content, this is on purpose. I wanted to have our engineers think about web performance at a high level so they can have a broader idea about which path they would go for when trying to optimize for speed.

**Comment by zlatanpham** [comment]
@monotykamary I think it's good to merge

**Pull request merged**